source/language/types.rst

```diff
@@ -222,6 +306,24 @@ namespace are listed in table `1 <#tab:real-constants>`__.
       | Euler’s number                | euler_gamma  | :math:`e`    | 2.7182818284...     |
       +-------------------------------+--------------+--------------+---------------------+
 
+Needs clarification:
+--------------------
+What is the default width of the my_const variable in the following
```

Comment:
We are thinking about making `const`s explicitly typed as well. Otherwise it has an implicit type of BigFloat/BigInt like constants in python.